src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataReader;
+import core.basesyntax.service.DataWriter;
+import core.basesyntax.service.GetListOfTransactions;
+import core.basesyntax.service.ReportCreator;
+import core.basesyntax.service.StorageUpdateService;
+import core.basesyntax.service.amount.BalanceHandlerImpl;
+import core.basesyntax.service.amount.OperationHandler;
+import core.basesyntax.service.amount.PurchaseHandlerImpl;
+import core.basesyntax.service.amount.SupplyHandlerImpl;
+import core.basesyntax.service.impl.DataReaderImpl;
+import core.basesyntax.service.impl.DataWriterImpl;
+import core.basesyntax.service.impl.GetListOfTransactionsImpl;
+import core.basesyntax.service.impl.ReportCreatorImpl;
+import core.basesyntax.service.impl.StorageUpdateServiceImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final DataReader DATA_READER = new DataReaderImpl();
```

Comment:
```suggestion
    private static final DataReader dataReader = new DataReaderImpl();
```
its not constants, just fields that are static just so we can call them in main
apply to other fields